### ISSUE TITLE
Tab/Window/Session Management?

### ISSUE THREAD

**Issue by arkag** [thmsbfft/oryoki#11]
I love the idea of a browser managed entirely by keyboard shortcuts, and I think adding a way to manage multiple windows opened or a tab system would be helpful. 

Maybe a tiling system instead? Something a la vim or tmux would be cool.

I'd appreciate this, but I can understand if this isn't within the scope of the project.

**Comment by thmsbfft** [comment]
Yeah, tiling is on the study list (see [readme](https://github.com/thmsbfft/oryoki#unsorted--study)). I'm also looking at picture-in-picture, though I'm not sure of the interest yet. Almost unrelated: have you heard of [Spectacle](https://www.spectacleapp.com/)? It kinda does that systemwide.

**Comment by arkag** [comment]
Lol, I don't know how I missed that in the readme... I'm not sure of the use of picture in picture, unless you can get youtube to fullscreen in your window and your window only. That would be super cool.

I hadn't heard of spectacle, but I use [hammerspoon](https://github.com/Hammerspoon/hammerspoon). It's handles windows with Lua scripting, and I can use auto focusing. Sadly, I haven't fixed the tiling I have scripted yet :(

**Comment by almosnow** [comment]
Just dropping my 2 ¢.

I really really really like how Oryoki handles fullscreen by keeping all the content within the window. I think this is possible because Oryoki is essentially chromeless (I always keep the title bar hidden), and I believe that this was designed intentionally this way.

I'm a little worried about how this idea of tabs or PiP is going to play along with chromeless windows. Perhaps they should only appear when you call the omnibox?
